.github/workflows/feature.yml

```diff
@@ -24,6 +24,8 @@ jobs:
       - name: Build
         run: |
           brew install automake
+          brew install autoconf
+          brew install libtool
```

Comment:
Could you please add these two rows into release.yaml file too ?

Reply:
Of course! This is my first time submitting a PR, so I'm still learning the process. Sorry for any inconvenience caused.Thank you for your help and guidance!

Reply:
Thank you for your contribution !